source/ui.js

```diff
@@ -5,11 +5,11 @@ const githubUrlFromGit = require('github-url-from-git');
 const isScoped = require('is-scoped');
 const util = require('./util');
 const git = require('./git-util');
-const {prereleaseTags, checkIgnoreStrategy} = require('./npm/util');
+const {prereleaseTags, checkIgnoreStrategy, getRegistryUrl} = require('./npm/util');
 const version = require('./version');
 const prettyVersionDiff = require('./pretty-version-diff');
 
-const printCommitLog = async repoUrl => {
+const printCommitLog = async (repoUrl, registryUrl) => {
```

Comment:
I wouldn't pass `registryUrl` as an argument to `printCommitLog`, as you should call `getRegistryUrl` inside the method.

Reply:
> I wouldn't pass `registryUrl` as an argument to `printCommitLog`, as you should call `getRegistryUrl` inside the method.

With that I would have to send two arguments (`options`, `pkg`) to use the `getRegistryUrl` inside the method. Would also be strange the `repoUrl` be an argument and `registryUrl` no.